### DISCUSSION TITLE
How to use the file_id got from get_chat method?

### DISCUSSION THREAD

**Discussion by GauthamramRavichandran** [python-telegram-bot/python-telegram-bot#2336] [Q&A] [answered]
Using the `get_chat` method, returns 4 file IDs if the user has set a profile pic.
I tried one of the ID with `send_photo`, it failed with '`type of file mismatch`'

How can I use this file ID to send this to user?

**Comment by Poolitzer** [comment, marked as answer]
I think we discussed this in the group. Anyway, according to docs, you can only uses these file_ids to download the pictures, not to send them.
